patch/common/attiny-0-series.yaml

```diff
@@ -0,0 +1,57 @@
+# This is probably good for at least ATtiny202/204/402/404/406, but may not be valid for ATtiny80* and ATtiny160*
+
+TCB0:
+  CTRLB:
+    _modify:
+      # 'async' is reserved
+      ASYNC:
+        name: "TCB_ASYNC"
```

Comment:
This is the only change that is hard _required_, as `async` is a reserved keyword.

Reply:
Technically it shouldn't be because newer versions of `svd2rust` will use a raw literal for such registers (`r#async`).  But I'm fine with changing it either way.

Reply:
(was previously discussed in #66 as well)

Reply:
Huh, I could have sworn I installed the newest version of `svd2rust` when I started on this, but it seems like I was on 0.17.

Although upgrading `svd2rust` (to either 0.18 or 0.19) fails (see #83)

Reply:
I see, I'll need to adjust the buildsystem to the new version...  But let's not block this PR on that - CI tested with a working revision of svd2rust and is happy.